tests/import_proxy_test.go

```diff
@@ -341,37 +455,58 @@ func getImporterPodIP(f *framework.Framework) string {
 	return podIP
 }
 
-func getProxyLog(f *framework.Framework) string {
+func getProxyLog(f *framework.Framework, since time.Time) string {
 	proxyPod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, proxyServerName, fmt.Sprintf("name=%s", proxyServerName))
 	Expect(err).ToNot(HaveOccurred())
 	fmt.Fprintf(GinkgoWriter, "INFO: Analyzing the proxy pod %s logs\n", proxyPod.Name)
-	log, err := RunKubectlCommand(f, "logs", proxyPod.Name, "-n", proxyPod.Namespace, "--since=15m")
+	log, err := RunKubectlCommand(f, "logs", proxyPod.Name, "-n", proxyPod.Namespace, fmt.Sprintf("--since-time=%s", since.Format(time.RFC3339)))
 	Expect(err).To(BeNil())
 	return log
 }
 
 func wasPodProxied(imgURL, podIP, proxyLog string, isHTTPS bool) bool {
```

Comment:
I know, all these changes are because these tests are bad and I am fixing them.

Reply:
But I did find a problem in this code, I think, so there will be a new push, so if there is something you want changed let me know

Reply:
It would be good to have a test to check if "wasPodProxied" can correctly match strings, it is not really complex, but it is a regExp so a single char can breake it. But do we even have a place for things like that?

Reply:
I am printing out the line that gets matched, so if a test fails on a match that is wrong it will print it. Not sure if it makes sense to have a test of a test?

Reply:
might be enough